lib/views/login/login_form.dart

```diff
@@ -12,23 +12,27 @@ class LoginForm extends ConsumerStatefulWidget {
 }
 
 class LoginFormState extends ConsumerState<LoginForm> {
+  final _loginFormKey = GlobalKey<FormState>();
   final _email = TextEditingController();
```

Comment:
```suggestion
  final _emailEditingController = TextEditingController();
```

Reply:
Let's keep it short with `_emailController`. :D What do you think?